memgraph_versioned_docs/version-2.3.1/connect-to-memgraph/drivers/php.md

```diff
@@ -89,8 +88,14 @@ You should see an output similar to the following:
 Node 1: Hello, World!
 ```
 
+_If you run apache/nginx instance you can open your project in web browser and you will get same result._
```

Comment:
```suggestion
```

We will add a bit better explanation of how to run and start the app once we merge this PR, so this will be obsolete.